frontend/src/metabase/admin/permissions/components/DataPermissionsHelp/DataPermissionsHelp.tsx

```diff
@@ -0,0 +1,126 @@
+import { t, jt } from "ttag";
+
+import { Accordion, Box, Flex, Stack, Text, Title } from "@mantine/core";
+import MetabaseSettings from "metabase/lib/settings";
+
+import { Icon } from "metabase/core/components/Icon";
+import ExternalLink from "metabase/core/components/ExternalLink";
+import { PermissionHelpDescription } from "metabase/admin/permissions/components/PermissionHelpDescription";
+
+export const DataPermissionsHelp = () => (
+  <Flex direction="column" py="1.375rem" px="1rem">
+    <Box px="0.75rem">
+      <Title order={4}>{t`Data permissions`}</Title>
+      <Text my="1rem">{t`People can be members of multiple groups, and Metabase grants them the most permissive level of access across all of a person's groups.`}</Text>
+    </Box>
+
+    <Accordion
+      chevron={<Icon name="chevrondown" size={12} />}
+      variant="separated"
+      defaultValue="database-level"
+    >
+      <Accordion.Item value="customization">
+        <Accordion.Control>Database levels</Accordion.Control>
+        <Accordion.Panel>
+          <Stack spacing="1rem" py="1rem">
+            <PermissionHelpDescription
+              icon="check"
+              iconColor="success"
+              name={t`Unrestricted`}
+              description={t`The group can use the query builder to ask questions of any table in the database.`}
+            />
+
+            <PermissionHelpDescription
+              icon="permissions_limited"
+              iconColor="warning"
+              name={t`Granular`}
+              description={t`The group can only use the query builder to ask questions of specific tables.`}
+            />
+
+            <PermissionHelpDescription
+              icon="database"
+              iconColor="warning"
+              name={t`Impersonated (Pro)`}
+              description={t`Impersonation associates a Metabase group with database-defined roles and their privileges. Metabase queries made by this group will respect the grants given to the database roles. You can use impersonation to give a group access to the native/SQL editor, while restricting the group's access to data based on a specific database role.`}
+            />
+
+            <PermissionHelpDescription
+              icon="eye_crossed_out"
+              iconColor="accent5"
+              name={t`No self-service`}
+              description={t`The group can't use the query builder or drill through existing questions. They also can't see the data in the Browse data section. They can still view questions based on this data, if they have permissions to the relevant collection.`}
+            />
+
+            <PermissionHelpDescription
+              icon="close"
+              iconColor="danger"
+              name={t`Block (Pro)`}
```

Comment:
I agree that block should have a (Pro) descriptor, but it was not there in the design docs. I assume you confirmed this with product already?
```suggestion
              name={t`Block`}
```